runtime/lua/vim/filetype.lua

```diff
@@ -1825,6 +1826,14 @@ local pattern = {
       return 'gitconfig'
     end
   end,
+  ['.*%.git/info/attributes'] = 'gitattributes',
+  ['.*/etc/gitattributes'] = 'gitattributes',
+  ['.*/%.config/git/attributes'] = 'gitattributes',
+  ['.*/git/attributes'] = function(path, bufnr)
+    if vim.env.XDG_CONFIG_HOME and path:find(vim.env.XDG_CONFIG_HOME .. '/git/attributes') then
```

Comment:
Not so sure about this. This makes the filetype detection dependent on an env var, and the file we are matching might not even be the file we are reading, i.e. if `path:find(vim.env.XDG_CONFIG_HOME .. '/git/attributes')` then any `.*/git/attirbutes` file will be detected.

Reply:
Sorry, I glossed over the vim version.

Ignore this.